src/services/users.services.js

```diff
@@ -0,0 +1,43 @@
+const { User } = require('../models/User.model');
+
+const getUsers = async () => {
+  const users = await User.findAll({ order: ['name'] });
+
+  return users;
+};
+
+const getUserById = async (id) => {
+  const user = await User.findByPk(id);
+
+  return user;
+};
+
+const createUser = async (name) => {
+  const user = await User.create({ name: name });
+
+  return user;
+};
+
+const updateUser = async (id, name) => {
+  await User.update({ name }, { where: { id } });
```

Comment:
Consider checking the result of the `update` operation to ensure that the update was successful. This can help identify cases where the user ID does not exist.